pkg/image/registry/image/strategy.go

```diff
@@ -38,8 +38,8 @@ func (s imageStrategy) PrepareForCreate(obj runtime.Object) {
 	if err := api.ImageWithMetadata(newImage); err != nil {
 		utilruntime.HandleError(fmt.Errorf("Unable to update image metadata for %q: %v", newImage.Name, err))
 	}
-
-	s.clearSignatureDetails(newImage)
+	// allow to set the signatures only through signatures subresource
```

Comment:
Why would I want to force this restriction?  Signatures aren't secret things, they are replayable by design.